nexus/src/external_api/console_api.rs

```diff
@@ -387,10 +387,10 @@ pub async fn consume_credentials(
             if let Some(referer) = &relay_state.referer {
                 referer.clone()
             } else {
-                "/".to_string()
+                "/orgs".to_string()
```

Comment:
This is okay in the short term but I'm a little reluctant to bake knowledge of console routes into Nexus.  I wonder if we shouldn't fix this instead by having the console provide the right redirect parameter in the first place.